src/test/java/org/terasology/flexiblepathfinding/LeapingJPSTest.java

```diff
@@ -0,0 +1,185 @@
+// Copyright 2022 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.flexiblepathfinding;
+
+import org.junit.jupiter.api.Test;
+import org.terasology.flexiblepathfinding.helpers.JPSTestHelper;
+import org.terasology.flexiblepathfinding.helpers.TestPaths;
+import org.terasology.flexiblepathfinding.helpers.TestWorlds;
+import org.terasology.flexiblepathfinding.plugins.basic.LeapingPlugin;
+
+/**
+ * Leaping movements are purely vertical (up) and require additional horizontal walking movements
+ * The following test cases only attempt to test that the leaping plugin alone does not incorrectly find valid paths for basic movements
+ **/
```

Comment:
Same clarification as above, plus the request to add one positive test case.

```suggestion
/**
 * Leaping describes purely vertical (up) paths and does not allow for horizontal steps.
 *
 * The following test cases (mostly) attempt to test that the leaping plugin alone does not incorrectly find valid paths that require horizontal steps.
 **/
```